packages/amplication-code-gen-types/src/plugin-events-params.ts

```diff
@@ -57,6 +57,17 @@ export interface CreateEntityControllerSpecParams extends EventParams {
   controllerId: namedTypes.Identifier;
   serviceId: namedTypes.Identifier;
 }
+
+export interface CreateUserInfoParams extends EventParams {
+  template: namedTypes.File;
+  templateMapping: { [key: string]: any };
+  filePath: string;
+}
+export interface CreateTokenPayloadInterfaceParams extends EventParams {
+  template: namedTypes.File;
+  templateMapping: { [key: string]: any };
```

Comment:
is it safe to make it so open ? with no defaults keys ?

Reply:
yes, this is also the standard for all other events. 
The idea is that the user can add params to the template and the mapping object